docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -1,118 +1,120 @@
 ## Install a package
 
-The [install](../reference/cli-reference/cli-ref-install.md) command downloads and installs a package into a project, defaulting to the current folder, using specified package sources. Install new packages into the *packages* folder in your project root directory.
+The NuGet CLI [install](../reference/cli-reference/cli-ref-install.md) command downloads and installs specified NuGet packages.
 
 > [!IMPORTANT]
-> The `install`command does not modify a project file or *packages.config*; in this way it's similar to `restore` in that it only adds packages to disk but does not change a project's dependencies. To add a dependency, either add a package through the Package Manager UI or Console in Visual Studio, or modify *packages.config* and then run either `install` or `restore`.
+> The `install` command doesn't modify the project file or *packages.config* file. The `install` and `restore` commands only add packages to disk, but don't add dependencies to projects. To add project dependencies, you must either:
+> 
+> - Add packages through the [Visual Studio Package Manager UI](install-use-packages-visual-studio.md) or [Package Manager Console](install-use-packages-powershell.md), or
+> - Modify [packages.config](../reference/packages-config.md) manually, then run `install` or `restore`.
+
+Use the `-OutputDirectory` option to install packages to a specific directory. If you don't specify an output directory, `install` uses the current directory.
 
-1. Open a command line and switch to the directory that contains your project file.
+```cli
+nuget install <packageID | configFilePath> -OutputDirectory <outputDirectory>
+```
 
-2. Use the following command to install a NuGet package to the *packages* folder.
+For example, to install the `Newtonsoft.json` package to the *packages* subdirectory, use the following command:
 
-    ```cli
-    nuget install <packageID> -OutputDirectory packages
-    ```
+```cli
+nuget install Newtonsoft.Json -OutputDirectory packages
+```
 
-    To install the `Newtonsoft.json` package to the *packages* folder, use the following command:
+Instead of specifying a package to install, you can specify an existing *packages.config* file in the current or another directory. The `install` command installs all the packages listed in the *packages.config* file.
 
-    ```cli
-    nuget install Newtonsoft.Json -OutputDirectory packages
-    ```
+```cli
+nuget install packages.config
+```
 
-Alternatively, you can use the following command to install a NuGet package using an existing `packages.config` file to the *packages* folder. This does not add the package to your project dependencies, but installs it locally.
+For example, the following command installs all the packages listed in the *packages.config* file at *C:\myPath* to the *packages* subdirectory in the current directory:
 
 ```cli
-nuget install packages.config -OutputDirectory packages
+nuget install C:\myPath\packages.config -OutputDirectory packages
```

Comment:
@v-thepet Just to confirm, looks like the full path is required for packages.config? @nkolev92 in case any clarification is needed.

Reply:
@Mikejo5000 - I'll experiment with using a relative path.

Reply:
Relative paths should work too.

Reply:
@Mikejo5000 @nkolev92 - Correct, relative paths work.